lib/command/CommandHandler.d.ts

```diff
@@ -0,0 +1,10 @@
+import { Base } from '../model';
+
+export default interface CommandHandler {
+  execute?(context: any): Base[];
+  revert?(context: any): Base[];
+  canExecute?(context: any): boolean;
+  preExecute?(context: any): void;
+  postExecute?(context: any): void;
+  $inject?: string[];
```

Comment:
`$inject` is necessary? Why?

Reply:
And if we require it it shall be `static`.

Reply:
I was asking myself that question. I'd like to communicate somehow that a command handler is going to be instantiated by the command stack and, therefore, _can_ use the `$inject` syntax. Unfortunately, interfaces do not support static members (cf. https://github.com/Microsoft/TypeScript/issues/13462).

Reply:
Let's not transport that through types, but through proper documentation.

You could consider to expand the `CommandHandler` documentation _or_ write new one (docs.bpmn.io).